switch option to gpl for experimenting

this is actually not needed anymore in this project so keeping it
for debugging purposes if we needed kprobes or similar again in future

i.e. this patch will not be applied by default - if needed,
use --force-nvidia-gpl-I-know-it-is-wrong option of patch.sh script
to apply this patch (another option will be needed to actually enable
it's effect in a build)

diff --git a/kernel/nvidia/nv.c b/kernel/nvidia/nv.c
index 656382c..24d703b 100644
--- a/kernel/nvidia/nv.c
+++ b/kernel/nvidia/nv.c
@@ -30,7 +30,11 @@
 
 
 
+#if !defined(FORCE_GPL_FOR_EXPERIMENTING)
 MODULE_LICENSE("NVIDIA");
+#else
+MODULE_LICENSE("GPL");
+#endif
 
 #endif
 #if defined(MODULE_INFO)
diff --git a/kernel/nvidia/nv-vgpu-vfio-interface.c b/kernel/nvidia/nv-vgpu-vfio-interface.c
index 4efb319..57923ff 100644
--- a/kernel/nvidia/nv-vgpu-vfio-interface.c
+++ b/kernel/nvidia/nv-vgpu-vfio-interface.c
@@ -270,7 +270,11 @@ NV_STATUS nvidia_vgpu_vfio_set_ops(vgpu_vfio_ops_t *ops)
     vgpu_vfio_ops = ops;
     return NV_OK;
 }
+#if !defined(FORCE_GPL_FOR_EXPERIMENTING)
 EXPORT_SYMBOL(nvidia_vgpu_vfio_set_ops);
+#else
+EXPORT_SYMBOL_GPL(nvidia_vgpu_vfio_set_ops);
+#endif
 
 NV_STATUS nvidia_vgpu_vfio_get_ops(rm_vgpu_vfio_ops_t *ops)
 {
@@ -293,7 +297,11 @@ NV_STATUS nvidia_vgpu_vfio_get_ops(rm_vgpu_vfio_ops_t *ops)
 
     return NV_OK;
 }
+#if !defined(FORCE_GPL_FOR_EXPERIMENTING)
 EXPORT_SYMBOL(nvidia_vgpu_vfio_get_ops);
+#else
+EXPORT_SYMBOL_GPL(nvidia_vgpu_vfio_get_ops);
+#endif
 
 static void revive_vfs(struct pci_dev *pci_dev)
 {
